### PR TITLE
GREEN-52125: Update webhook retry policy

### PR DESCRIPTION
**Pull request by OmodaraAdonai**
https://greenhouseio.atlassian.net/browse/GREEN-52125

### CODE REVIEW

File: source/includes/webhooks/_introduction.md [hunk 1]
```diff
@@ -188,14 +188,14 @@ If a webhook is disabled, it will not trigger when the event occurs. Webhooks be
...
 
-In the event of a failed webhook request (due to timeout, a non HTTP 200 response, or network issues), Greenhouse will attempt a maximum of 6 retries according to the formula on the right:
+In the event of a failed webhook request (due to timeout, a non HTTP 200 response, or network issues), Greenhouse will make up to 6 attempts according to the formula on the right.
 
 This formula increases the amount of time between each retry, while assigning a random number of seconds to avoid consistent failures from overload or contention.
 
-Greenhouse will attempt 6 retries over the course of 15 hours.
+Greenhouse will make up to 6 attempts over the course of 7 hours.
 
 ```ruby
```
Comment (jsurrett) [inline source/includes/webhooks/_introduction.md:197]: Do we even need this part? I would suggest removing the code section.

File: source/includes/webhooks/_introduction.md [hunk 2]
```diff
@@ -205,13 +205,13 @@ sidekiq_retry_in do |index, _exception|
...
-|       5      |      240m     |        7h 16m      |
-|       6      |      480m     |       15h 16m      |
+| Attempt number | Next Attempt in | Total waiting time  |
+| -------------- | --------------- | ------------------  |
+|       1        |        1m       |            0m       |
+|       2        |       15m       |        0h  1m       |
+|       3        |       60m       |        0h 16m       |
+|       4        |      120m       |        1h 16m       |
+|       5        |      240m       |        3h 16m       |
+|       6        |        --       |        7h 16m       |
```
Comment (jsurrett) [inline source/includes/webhooks/_introduction.md:217]: Would it make more sense to put the last column first?
```suggestion
| Approx. Total waiting time | Attempt number | Next Attempt in | 
| -------------- | --------------- | ------------------  |
|   0m         |   1        |        1m       | 
|   0h  1m   |   2        |       15m       |       
|   0h 16m  |   3        |       60m       |        
|   1h 16m   |   4        |      120m       |        
|   3h 16m  |   5        |      240m       | 
|   7h 16m  |   6        |        --       |  
```

File: source/includes/webhooks/_introduction.md (hunk 1 above)
Comment (OmodaraAdonai) [reply source/includes/webhooks/_introduction.md]: Thanks, done

File: source/includes/webhooks/_introduction.md (hunk 2 above)
Comment (OmodaraAdonai) [reply source/includes/webhooks/_introduction.md]: Thanks, this is done also!

**Review by jsurrett — APPROVED** [review]

**Review by jlou-gh — APPROVED** [review]

**Review by jsurrett — APPROVED** [review]

**Review by anirudhmungre — APPROVED** [review]

**Review by julienguyenn — APPROVED** [review]

**Review by jlou-gh — APPROVED** [review]

**Review by jlou-gh — APPROVED** [review]

**Review by anirudhmungre — APPROVED** [review]

**Review by julienguyenn — APPROVED** [review]

**Pull request merged**